readme (#75)
glklimmer/warppcs: README.md

# About WARPPC

> Warppc (Placehold Name) is a 2D multiplayer game written in Rust and [Bevy](https://bevyengine.org/).

Reviewer: ```suggestion
Warppc (project name) is a 2D multiplayer game written in Rust and [Bevy](https://bevyengine.org/).
```

> Warppc (Placehold Name) is a 2D multiplayer game written in Rust and [Bevy](https://bevyengine.org/).

Reviewer: ```suggestion
Warppc (Placehold Name) is a 2D multiplayer game written in Rust using the [Bevy](https://bevyengine.org/) engine.
```

We are currently programming our [MVP](https://github.com/glklimmer/warppcs/milestone/1), so stay tuned !

![til](./prototype.gif)

## Game Loop
Our planned game loop consist of:

> - Slay NPC to gather loot

Reviewer: ```suggestion
- Slay NPCs to gather loot
```
